xml/art_sle_ha_install_quick.xml

```diff
@@ -73,10 +73,7 @@
     </listitem>
     <listitem>
      <para>A shared storage device, used as SBD fencing mechanism.
-     <remark>toms 2016-08-01: Split-brain resolution via SBD.
-      (lars): Maybe be mentioned here, to make it more visible that our
-      product does support this?
-     </remark>
+      This solves split brain scenarios as well.
```

Comment:
Instead of "solves" perhaps better: "avoids": This avoids split brain scenarios. ("as well" is not needed as avoiding split brain scenarios and data corruption is the whole point why fencing is used).